src/main/java/com/internet/shop/web/filters/AuthenticationFilter.java

```diff
@@ -23,11 +29,11 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         HttpServletRequest req = (HttpServletRequest) request;
         HttpServletResponse resp = (HttpServletResponse) response;
         String url = req.getServletPath();
-        if (url.equals("/login") || url.equals("/registration")) {
+        if (isAccessesible(url)) {
```

Comment:
this can be replaced with `contains()`

Reply:
Replaced to contains.

Reply:
But i replaced it inside of isAccessesible().